packages/plugin-ext/src/main/browser/view/plugin-view-registry.ts

```diff
@@ -14,32 +14,32 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
-import { injectable, inject, postConstruct } from 'inversify';
 import {
-    ApplicationShell, ViewContainer as ViewContainerWidget, WidgetManager,
-    ViewContainerIdentifier, ViewContainerTitleOptions, Widget, FrontendApplicationContribution,
-    StatefulWidget, CommonMenus, BaseWidget
-} from '@theia/core/lib/browser';
-import { ViewContainer, View } from '../../../common';
-import { PluginSharedStyle } from '../plugin-shared-style';
-import { DebugWidget } from '@theia/debug/lib/browser/view/debug-widget';
-import { PluginViewWidget, PluginViewWidgetIdentifier } from './plugin-view-widget';
-import { SCM_VIEW_CONTAINER_ID, ScmContribution } from '@theia/scm/lib/browser/scm-contribution';
-import { EXPLORER_VIEW_CONTAINER_ID } from '@theia/navigator/lib/browser';
-import { FileNavigatorContribution } from '@theia/navigator/lib/browser/navigator-contribution';
-import { DebugFrontendApplicationContribution } from '@theia/debug/lib/browser/debug-frontend-application-contribution';
-import { Disposable, DisposableCollection } from '@theia/core/lib/common/disposable';
-import { CommandRegistry } from '@theia/core/lib/common/command';
-import { MenuModelRegistry } from '@theia/core/lib/common/menu';
```

Comment:
@owlJaeger can you revert these changes, they are unnecessary and are noisy during reviews.

Reply:
Sure, will take care of it.

Reply:
Done

Reply:
Done